website/docs/docs/build/documentation.md

```diff
@@ -24,13 +27,16 @@ import CopilotBeta from '/snippets/_dbt-copilot-avail.md';
 
 ## Overview
 
-dbt provides a way to generate documentation for your dbt project. The documentation for your project includes:
+dbt provides a scalable way to [generate](#generating-documentation) documentation for your dbt project using descriptions and commands. The documentation for your project includes:
 * **Information about your project**: including model code, a DAG of your project, any tests you've added to a column, and more.
 * **Information about your <Term id="data-warehouse" />**: including column data types, and <Term id="table" /> sizes. This information is generated by running queries against the information schema.
+* Importantly, dbt also provides a way to add **descriptions** to models, columns, sources, and more, to further enhance your documentation.
 
-Importantly, dbt also provides a way to add **descriptions** to models, columns, sources, and more, to further enhance your documentation.
+The following sections describe how to [add descriptions](#adding-descriptions-to-your-project) to your project, [generate documentation](#generating-documentation), how to use [docs blocks](#using-docs-blocks), and set a [custom overview](#setting-a-custom-overview) for your documentation.
 
 ## Adding descriptions to your project
+
+Before generating documentation, you'll need to add [descriptions](/reference/resource-properties/description) to your project resources.
```

Comment:
We could combine the two to increase readability:
```suggestion
Before generating documentation, add [descriptions](/reference/resource-properties/description) to your project resources. Add the `description:` key to the same YAML files where you declare [tests](/docs/build/data-tests). For example:
```

Reply:
ooo nice sugg!